source/_components/sensor.oasa_telematics.markdown

```diff
@@ -0,0 +1,72 @@
+---
+layout: page
+title: "OASA Telematics"
+description: "Instructions on how to integrate bus and trolley arrival data for Greek OASA Telematics within Home Assistant."
+date: 2019-03-05 21:52
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: oasa.png
+ha_category:
+  - Transport
+  - Sensor
+ha_iot_class: Cloud Polling
+ha_release: 0.91
+---
+
+The `oasa_telematics` sensor will provide you with bus and trolley arrival times for Greek public transport for Athens, using real-time data from [OASA Telematics](http://telematics.oasa.gr/en/).
+
+## {% linkable_title Configuration %}
+
+Add a sensor to your `configuration.yaml` file as shown in the example:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: oasa_telematics
+    route_id: 'YOUR_ROUTE_ID'
```

Comment:
Must this between `' '` ?

Reply:
Quotes are not needed for `route_id` indeed.

Reply:
And also for the `stop_id`?

Reply:
No, `stop_id` is a number that might have leading zeroes (like in the example I gave), so that needs to be a string.